src/components/pages/Community/Community.js

```diff
@@ -17,6 +24,36 @@ class Community extends Component {
             <img src={chevronDown} alt="down"/>
           </button>
         </div>
+        <div className="community-events-container">
+          <p className="section-label">Events</p>
+          <h2>What's happening</h2>
+          <SectionToggle
+            toggleLabel1="Upcoming"
+            toggleLabel2="Past"/>
+          <div className="event-card">
+            <div className="event-info">
+              <p className="event-label">Information Architecture</p>
+              <h2 className="event-name">UX&Chill Ep 08</h2>
+
+              <ul className="event-details">
+                <img src={calendar} alt="calender"/><li>24 September 2018, 5:00 to 7:30 PM</li>
```

Comment:
This should be temporarily be a prop. We may need to discuss how we will record event start and end times in the database.